### PR TITLE
Update install_add_to_your.ini + manual_change.ngc

### PR DESCRIPTION
**Pull request by alkabal**
Hi

I know for now you want to keep the G38.4 but this allow to view other change more easily.

Br

### CODE REVIEW

**Review by freisei — COMMENTED** [review]
currently we are trying to implement G38.4 - but these are removed here.
G0 makes sense for going to non-critical positionions it´s exactly what it is made for. Why did you change this?

**Review by freisei — COMMENTED** [review]
could be useful if you tell us why you want to do your changes.

File: macros/psng_manual_change.ngc [hunk 1]
```diff
@@ -17,11 +19,12 @@ O100     return [999]
 O100 endif
 
 ;first go up
+F#<_ini[TOOLSENSOR]RAPID_SPEED>
 G90
-G53 G0 Z[#<_ini[AXIS_Z]MAX_LIMIT>-0.1]
+G53 G1 Z[#<_ini[AXIS_Z]MAX_LIMIT>-1]
```
Comment (kiall) [inline macros/psng_manual_change.ngc:24]: Why change from -0.1 to -1? The goal is is to get as high as possible to reduce the risk of making contact with the part.

File: macros/psng_manual_change.ngc [hunk 2]
```diff
@@ -17,11 +19,12 @@ O100     return [999]
...
 ; then move to change position
-G53 G0 X[#<_ini[CHANGE_POSITION]X>] Y[#<_ini[CHANGE_POSITION]Y>]
-G53 G0 Z[#<_ini[CHANGE_POSITION]Z>]
+G53 G1 X[#<_ini[CHANGE_POSITION]X>] Y[#<_ini[CHANGE_POSITION]Y>]
+G53 G1 Z[#<_ini[CHANGE_POSITION]Z>]
```
Comment (kiall) [inline macros/psng_manual_change.ngc:27]: Switching to G1 means the rapid override in the UI no longer works, and at least with my configuration, will make the probe maneuvers far slower. Why would G1 be preferred here?

File: macros/psng_manual_change.ngc [hunk 3]
```diff
@@ -46,20 +49,21 @@ O400 endif
...
 O500 if [#5070 EQ 0]
 G90
 O500 return [-3] ; indicate probe contact failure to epilog
 O500 endif
 
+F#<_ini[TOOLSENSOR]RAPID_SPEED>
 G90
-G53 G0 Z[#<_ini[CHANGE_POSITION]Z>]
+G53 G1 Z[#<_ini[CHANGE_POSITION]Z>]
+G53 G1 X[#<_ini[CHANGE_POSITION]X>] Y[#<_ini[CHANGE_POSITION]Y>]
```
Comment (kiall) [inline macros/psng_manual_change.ngc:66]: Why move back to the change position? Given this isn't the same position as we were at when we entered the macro, there doesn't seem to be much value in moving back here.

**Comment by alkabal** [comment]
> Why move back to the change position? Given this isn't the same position as we were at when we entered the macro, there doesn't seem to be much value in moving back here.

I'm sorry you are right the machine need to stay at this position !

> Switching to G1 means the rapid override in the UI no longer works, and at least with my configuration, will make the probe maneuvers far slower. Why would G1 be preferred here?

At the end i agree with you, i do this because i have set my machine max vel too much high so i need to decrease that for having a acceptable G0.

File: macros/psng_manual_change.ngc (hunk 1 above)
Comment (alkabal) [reply macros/psng_manual_change.ngc:24]: you are right it's only because dislike to be too close to the sensor, but this is ok with 0.1

**Comment by alkabal** [comment]
> could be useful if you tell us why you want to do your changes.
> What is the sense of the empty line?

Only because github report this as error...

**Comment by freisei** [comment]
> Only because github report this as error...
Where?

**Comment by alkabal** [comment]
https://github.com/linuxcnc-probe-screen/probe-screen-ng/pull/13/commits/ae48fc22b2cbc6ec05fc8fe1136f589ce683d6d9


Generally linuxcnc and other linux editable file like to have a empty line at the end.

**Comment by freisei** [comment]
ah OK, thanks for this input.

**Review by freisei — APPROVED** [review]

**Review by freisei — CHANGES_REQUESTED** [review]
is it possible to approve only one commit of these 2?

**Comment by kiall** [comment]
> is it possible to approve only one commit of these 2?

No, it's the whole PR or nothing :)

**Comment by alkabal** [comment]
you can add yoursefl the empty line with next update (this is not hurry) and i can close this one

**Comment by freisei** [comment]
ok, than its smart to do only small or atomic pull requests.

**Pull request closed without merging**